habitat-lab/habitat/articulated_agents/humanoids/kinematic_humanoid.py

```diff
@@ -31,7 +31,7 @@ def _get_humanoid_params(self):
             wheel_mtr_vel_gain=None,
             wheel_mtr_max_impulse=None,
             ee_offset=[mn.Vector3(), mn.Vector3()],
-            ee_links=[14, 18],
+            ee_links=[20, 39],
```

Comment:
Was this causing NANs when animating?

Reply:
Sorry for the delay. Is this causing Nans?